src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/TagHelperResultCache.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I tried to build this class so it could be used by both the `OOPTagHelperResolver` (devenv) AND the `RemoteTagHelperDeltaProvider` (ServiceHub) since they both need to cache information/calculate or apply deltas